analytic_journal/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2015 Odoo SA
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```

Comment:
```suggestion
```
Delete this line in all files, can't add copyright here because it was part of Odoo core